### PR TITLE
Add Isidora documenation page

### PR DESCRIPTION
**Pull request by gonzaponte**
Add documentation for Isidora.

https://pr006-sw-docs.readthedocs.io/en/latest/isidora.html

### CODE REVIEW

File: docs/source/isidora.rst [hunk 1]
```diff
@@ -1,5 +1,54 @@
 Isidora
 ==========
 
-.. caution::
-  This page is currently not created, we are working on it. If you would like to contribute on this documentation, reach `me <helena.almamol@gmail.com>`_!
+*From ancient Greek, Ἰσίδωρος: gift of Isis.*
+
+Due to the bias configuration of the PMTs, the PMT waveform does not represent the actual signal produced by the PMT, but its derivative. This effect is treated online (for trigger) and offline (for data processing) by means of the BaseLine Restoration (BRL) algorithm. *Isidora* processes **RWF**\ s coming from either the *Decoder* (detector) or from *Diomira* (MC) and applies the BLR algorithm to the waveforms. The output are the so-called Corrected Waveforms, or **CWF**.
```
Comment (halmamol) [inline docs/source/isidora.rst:6]: I would add a line saying where these CWF are used, or if they can be the input of any city (not sure, in fact).

File: docs/source/isidora.rst [hunk 2]
```diff
@@ -1,5 +1,54 @@
...
+ * ``/RD/pmtrd``
+ * ``/RD/sipmrd``
+
+.. _Isidora output:
+
+Output
+------
+
+ * ``/BLR/pmtcwf``: the PMT corrected waveforms
+ * ``/BLR/sipmrwf``: the SiPM raw waveforms (WTF? why are they copied?)
```
Comment (halmamol) [inline docs/source/isidora.rst]: Maybe review this line :P

File: docs/source/isidora.rst (hunk 1 above)
Comment (gonzaponte) [reply docs/source/isidora.rst:6]: No, CWFs are not used anywhere else. That's the reason for the note that follows. Any suggestions on how to make it better without repetition?

Comment (halmamol) [reply docs/source/isidora.rst:6]: You're right. I would leave it as it is. Maybe just add some bold in 'debugging' on the note. (Btw, I resolved the other comment I left on the review, don't forget to add it!)

**Review by halmamol — APPROVED** [review]
This PR includes Isidora documentation. I approve it! Good job, again, @gonzaponte 😄  We need two reviewers to be merged though. Could you take a quick look @paolafer?

**Review by paolafer — APPROVED** [review]
Simple documentation, straight to the point. Approved!

**Pull request merged**